mne/stats/cluster_level.py

```diff
@@ -921,6 +911,7 @@ def get_progress_bar(seeds):
                         'cluster%s to exclude from subsequent iterations'
                         % (n_step_downs, n_removed, a_text,
                            _pl(n_removed)))
+        pb.cleanup()
```

Comment:
Unfortunately no we can't, as when one `ProgressBar` within a parallel process completed, it would GC and then delete the file that the other instances need.

I'll try making `ProgressBar` a context manager, and in the main thread/process that creates it do `with pb:` that will cause a cleanup at the end.